src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1430,6 +1438,20 @@
                         </UserData>
                     </template>
 
+                    <template tid="InitializeYieldProcessorNormalized">
+                        <data name="YieldsPerNormalizedYield" inType="win:UInt32" />
+                        <data name="OptimalMaxNormalizedYieldsPerSpinIteration" inType="win:UInt32" />
```

Comment:
The values computed in .NET Framework are different. I suggest sending just the `nsPerYield` value as an 8-byte float, that is commonly measured in .NET Framework and .NET Core, and the rest of the values can be calculated from it.